### ISSUE TITLE
On data with mostly just routes data (no bgp or ospf), table show gives Exception

### ISSUE THREAD

**Issue by jopietsch** [netenglabs/suzieq#297]
**Describe the bug**
```
(suzieq-QZqWZG2D-py3.8) jpietsch@jpietsch-server:~/suzieq$ suzieq/cli/suzieq-cli table show
Error running command: "['vlanName' 'state'] not found in axis"
------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jpietsch/.cache/pypoetry/virtualenvs/suzieq-QZqWZG2D-py3.8/src/python-nubia/nubia/internal/cmdbase.py", line 482, in run_cli
    return fn(**kwargs)
  File "/home/jpietsch/suzieq/suzieq/cli/sqcmds/TableCmd.py", line 41, in show
    df = self.sqobj.get(hostname=self.hostname, namespace=self.namespace)
  File "/home/jpietsch/suzieq/suzieq/sqobjects/tables.py", line 36, in get
    info.update(table_obj.get_table_info(
  File "/home/jpietsch/suzieq/suzieq/engines/pandas/engineobj.py", line 124, in get_table_info
    all_time_df = self.get_valid_df(table, view='all', **kwargs)
  File "/home/jpietsch/suzieq/suzieq/engines/pandas/engineobj.py", line 101, in get_valid_df
    table_df.drop(columns=drop_cols, inplace=True)
  File "/home/jpietsch/.cache/pypoetry/virtualenvs/suzieq-QZqWZG2D-py3.8/lib/python3.8/site-packages/pandas/core/frame.py", line 4167, in drop
    return super().drop(
  File "/home/jpietsch/.cache/pypoetry/virtualenvs/suzieq-QZqWZG2D-py3.8/lib/python3.8/site-packages/pandas/core/generic.py", line 3887, in drop
    obj = obj._drop_axis(labels, axis, level=level, errors=errors)
  File "/home/jpietsch/.cache/pypoetry/virtualenvs/suzieq-QZqWZG2D-py3.8/lib/python3.8/site-packages/pandas/core/generic.py", line 3921, in _drop_axis
    new_axis = axis.drop(labels, errors=errors)
  File "/home/jpietsch/.cache/pypoetry/virtualenvs/suzieq-QZqWZG2D-py3.8/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 5284, in drop
    raise KeyError(f"{labels[mask]} not found in axis")
KeyError: "['vlanName' 'state'] not found in axis"
------------------------------------------------------------
```

**Comment by ddutt** [comment]
Fixed by one of the many bug fixes checked in. I can't get this to happen now. Tested with a routes only DB.

**Comment by jopietsch** [comment]
I don't think this is fixed, but I can't be sure. I was referring to a specific set of data that we have that only has route information, and it's older suzieq data. right now with that dataset I always see https://github.com/netenglabs/suzieq/issues/301, so I can't be sure that this is fixed or not.

**Comment by ddutt** [comment]
This should now be fixed with the fix for #301.
